app/uk/gov/hmrc/leakdetection/config/AppConfig.scala

```diff
@@ -65,31 +65,37 @@ class AppConfigProvider @Inject()(
 
       alerts =
         Alerts(SlackConfig(
-          enabled             = configuration.get[Boolean    ]("alerts.slack.enabled"),
-          adminChannel        = configuration.get[String     ]("alerts.slack.adminChannel"),
-          defaultAlertChannel = configuration.get[String     ]("alerts.slack.defaultAlertChannel"),
-          username            = configuration.get[String     ]("alerts.slack.username"),
-          iconEmoji           = configuration.get[String     ]("alerts.slack.iconEmoji"),
-          alertChannelEnabled  = configuration.get[Boolean    ]("alerts.slack.alertChannel.enabled"),
-          repositoryChannelEnabled  = configuration.get[Boolean    ]("alerts.slack.repositoryChannel.enabled"),
-          messageText         = configuration.get[String     ]("alerts.slack.messageText"),
-          leakDetectionUri    = configuration.get[String     ]("alerts.slack.leakDetectionUri"),
-          warningText         = configuration.get[String     ]("alerts.slack.warningText"),
-          warningsToAlert     = configuration.get[Seq[String]]("alerts.slack.warningsToAlert")
-        ))
+          enabled                  = configuration.get[Boolean    ]("alerts.slack.enabled"),
+          adminChannel             = configuration.get[String     ]("alerts.slack.adminChannel"),
+          defaultAlertChannel      = configuration.get[String     ]("alerts.slack.defaultAlertChannel"),
+          username                 = configuration.get[String     ]("alerts.slack.username"),
+          iconEmoji                = configuration.get[String     ]("alerts.slack.iconEmoji"),
+          alertChannelEnabled      = configuration.get[Boolean    ]("alerts.slack.alertChannel.enabled"),
+          repositoryChannelEnabled = configuration.get[Boolean    ]("alerts.slack.repositoryChannel.enabled"),
+          messageText              = configuration.get[String     ]("alerts.slack.messageText"),
+          leakDetectionUri         = configuration.get[String     ]("alerts.slack.leakDetectionUri"),
+          warningText              = configuration.get[String     ]("alerts.slack.warningText"),
+          warningsToAlert          = configuration.get[Seq[String]]("alerts.slack.warningsToAlert")
+        )),
+      timeoutBackoffMillis         = configuration.getMillis("queue.timeoutBackOff"),
```

Comment:
`configuration.get[Duration]` then the type will define what it is, rather than requiring the unit in the name